src/Makevars.win

```diff
@@ -0,0 +1,12 @@
+PKG_CPPFLAGS = -DARMA_64BIT_WORD=1 -I.
+PKG_CXXFLAGS = $(SHLIB_OPENMP_CXXFLAGS)
+CXX = /usr/local/opt/llvm/bin/clang++
```

Comment:
In reference to the discussion/last response of DIrk Eddelbuettel's suggestion in [this](https://stackoverflow.com/questions/32974568/r-package-with-c-c-and-openmp-how-to-make-makevars-file-under-mypackage-sr) post, is there a way to not assume "clang-ANYTHING"? This seems to be a requirement for portability, i.e. CRAN.